docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -70,7 +74,9 @@ curl -sSLf https://get.openziti.io/tun/scripts/install-ubuntu.bash | bash
 | 10 Buster   | Bionic 18.04 | x86_64        |
 |  9 Stretch  | Xenial 16.04 | x86_64        |
 
-This example subscribes you to the Ubuntu `focal` repo which will work well in most cases. Alternatively, you may refer to the table to find the Ubuntu release name that is the contemporary of your Debian release. Then, substitute the Ubuntu release name for `focal` in the `/etc/apt/sources.list.d/openziti.list` file.
+This example subscribes to the Ubuntu `focal` repo. Refer to the table to find the Ubuntu release name that is the
```

Comment:
Is it 'an example' Can we make this any easier by having the user set a variable and then run the command? looking at the command, it's very difficult to find the "thing" I'd need to do. Also, why is "installing the RPM" in that tab? Shouldn't it be at the same level as Ubuntu/Debian???